nodejs/awsx/elasticloadbalancingv2/loadBalancer.ts

```diff
@@ -42,7 +46,18 @@ export abstract class LoadBalancer extends pulumi.ComponentResource {
             securityGroups: this.securityGroups.map(g => g.id),
             tags: utils.mergeTags(args.tags, { Name: longName }),
         }, parentOpts);
-   }
+    }
+
+    public attachTarget(
+            name: string,
+            args: pulumi.Input<LoadBalancerTarget> | LoadBalancerTargetProvider | aws.ec2.Instance,
```

Comment:
note @lukehoban I'm hardcoding in aws.ec2.Instance here (but i don't like it).   The goal is to easily allow clients to just call `lb.attachTarget(ec2Instance)`.  However, that won't work because ec2Instance doesn't implement LoadBalancerTargetProvider.  I *could* make it implement that interface by using mixins in the aws package.  But that would be a first where we do that.  

I'm curious what you think here.  

1. Should we just recognize this super common type, like i've done here?
2. Should we require the user to call `new awsx.elasticloadbalancingv2.Ec2InstanceTarget` themselves from code?
3. Should I go monkeypatch tihs interface into `aws`?

Something else?

Reply:
I personally think this is okay as is.  I think in practice it will be okay to handle the universe of things that can be load balancer targets in one place here.

Reply:
ok, sounds good.